examples/traefik-v2/docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
             - ENABLE_HSTS
             - ENABLE_XMPP_WEBSOCKET
             - DISABLE_HTTPS
+            - DISABLE_FLOC
```

Comment:
Please don't add it here, the traefik files will be moved out

Reply:
Done in 9c5a4ac